### DISCUSSION TITLE
Why isn't per device polling showing all devices?

### DISCUSSION THREAD

**Discussion by Mastiffen** [zwave-js/zwave-js-ui#2180] [Q&A] [answered]
I want to be able to poll my thermostats more often than they send info (that only happens when there's a change in the value). But when I open the polling window in General settins, New Value, I can only see 8 of my 30+ devices. I can see all the thermostats in the info that's sendt to Hass, and they are marked as present in the control panel, but all but one are missing from the polling drop-down. It seems like it's only showing one each of all the different types of devices (one smoke alarm, one thermostat, one switch of one type, one switch of another type and so on). Is that a limitation in the polling setup, or is there something wrong with my system?

**Comment by kpine** [comment, marked as answer]
The menu shows all of the unique device models. You pick the model and it polls all of the nodes matching that model. It's not showing individual nodes.

**Reply by Mastiffen** [reply]
Aha, thanks! Then I misunderstood the concept of "per device polling".

**Reply by kpine** [reply]
"Device" is an overloaded term, for sure.

**Reply by Mastiffen** [reply]
Yeah, it fooled me. I was thinking devices as in each separate thing.
